fabric-convention-tags-v1/src/main/java/net/fabricmc/fabric/api/tag/convention/v1/ConventionalBlockTags.java

```diff
@@ -33,7 +33,21 @@ public class ConventionalBlockTags {
 	public static final TagKey<Block> BOOKSHELVES = register("bookshelves");
 	public static final TagKey<Block> GLASS_BLOCKS = register("glass_blocks");
 	public static final TagKey<Block> GLASS_PANES = register("glass_panes");
+
+	public static final TagKey<Block> YELLOW_SANDSTONE_BLOCKS = register("yellow_sandstone_blocks");
+	public static final TagKey<Block> RED_SANDSTONE_BLOCKS = register("red_sandstone_blocks");
+	public static final TagKey<Block> SANDSTONE_BLOCKS = register("sandstone_blocks");
+	public static final TagKey<Block> YELLOW_SANDSTONE_SLABS = register("yellow_sandstone_slabs");
+	public static final TagKey<Block> RED_SANDSTONE_SLABS = register("red_sandstone_slabs");
+	public static final TagKey<Block> SANDSTONE_SLABS = register("sandstone_slabs");
+	public static final TagKey<Block> YELLOW_SANDSTONE_STAIRS = register("yellow_sandstone_stairs");
+	public static final TagKey<Block> RED_SANDSTONE_STAIRS = register("red_sandstone_stairs");
+	public static final TagKey<Block> SANDSTONE_STAIRS = register("sandstone_stairs");
+
+	public static final TagKey<Block> JOB_SITE_BLOCKS = register("job_site_blocks");
```

Comment:
Isn't there already `#minecraft:acquirable_job_site` for this?

Reply:
`#minecraft:acquirable_job_site` is a POI tag, not a block tag. You could theoretically use it to get the blocks, however it's not very efficent

Reply:
Ah, nevermind then. I think a better name would be `#c:villager_job_sites` to explain that it's about villagers. The `_blocks` suffix is unnecessary here imo.